tests/ingest/ingest_with_text_embedding_processor.yaml

```diff
@@ -0,0 +1,108 @@
+$schema: ../json_schemas/test_story.schema.yaml
+
+skip: false
+description: |
+  This test story checks that we can create an ingest pipeline with a text 
+  embedding processor a uses it to create an index.
+epilogues:
+  - path: /_ingest/pipeline/books_pipeline
+    method: DELETE
+    status: [200, 404]
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+  # - path: /_plugins/_ml/model_groups/{model_group_id}
+  #   method: DELETE
+  #   status: [200, 404]
+  #   parameters_from_environment:
+  #     model_group_id: test_model_group_id
+  - path: /_plugins/_ml/models/{model_id}
+    method: DELETE
+    status: [200, 404]
+    parameters_from_environment:
```

Comment:
We should support a syntax that resolves from multiple sources:

```yaml
parameters:
   model_id: "${env.test_model_id}"
```

Note though that if the model could not be created this cleanup doesn't work. Is there a way to delete all models or specify the model ID in the request when it gets created?